docs/public/installing-odo.adoc

```diff
@@ -46,28 +46,28 @@ The following section describes how to install `odo` on different platforms via
 
 === Binary installation
 
-. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo.exe`] file.
-. Add the location of your `odo.exe` to your `GOPATH/bin` directory.
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo-windows-amd64.exe`] file.
+. Rename the downloaded file to `odo.exe` and move it to a folder of your choice, for example `C:\odo`
+. Add the location of your `odo.exe` to your `%PATH%`.
 
 === Setting the `PATH` variable for Windows 10
 
 Edit `Environment Variables` using search:
+The following examples demonstrates how to set up a path variable. Your binaries can be located in any location, but this example uses C:\odo as the location.
 
 . Click *Search* and type `env` or `environment`.
 . Select *Edit environment variables for your account*.
 . Select *Path* from the *Variable* section and click *Edit*.
-. Click *New* and type `C:\go-bin` into the field or click *Browse* and select the directory, and click *OK*.
+. Click *New* and type `C:\odo` into the field or click *Browse* and select the directory, and click *OK*.
 
 === Setting the `PATH` variable for Windows 7/8
 
-The following example demonstrates how to set up a path variable. Your binaries can be located in any location, but this example uses C:\go-bin as the location.
-
-. Create a folder at `C:\go-bin`.
-. Right click *Start* and click *Control Panel*.
-. Select *System and Security* and then click *System*.
-. From the menu on the left, select the *Advanced systems settings* and click the *Environment Variables* button at the bottom.
-. Select *Path* from the *Variable* section and click *Edit*.
-. Click *New* and type `C:\go-bin` into the field or click *Browse* and select the directory, and click *OK*.
+. Click *Start* and in the `Search` box types `Advance System Settings`.
+. Select *Advanced systems settings* and click the *Environment Variables* button at the bottom.
```

Comment:
`Advanced System Settings` maybe? I'm not sure since I don't have a Windows box handy.